CHANGELOG.md

```diff
@@ -20,6 +20,9 @@
   * `prometheus_sd_refresh_duration_seconds` renamed to `cortex_prometheus_sd_refresh_duration_seconds`
 * [CHANGE] Query-frontend: the default value for `-query-frontend.not-running-timeout` has been changed from 0 (disabled) to 2s. The configuration option has also been moved from "experimental" to "advanced". #7126
 * [CHANGE] Store-gateway: to reduce disk contention on HDDs the default value for `blocks-storage.bucket-store.tenant-sync-concurrency` has been changed from `10` to `1` and the default value for `blocks-storage.bucket-store.block-sync-concurrency` has been changed from `20` to `4`. #7136
+* [CHANGE] Server: `server.Config.ReportGRPCCodesInInstrumentationLabel` has been set to true, making ingesters report gRPC status codes as `status_code` labels in the `cortex_request_duration_seconds` metric by default. #7144
```

Comment:
I would remove the "`server.Config.ReportGRPCCodesInInstrumentationLabel` has been set to true" part because it's an internal thing. I would just explain the change from the user perspective. You could replace it with the mention to `-server.report-grpc-codes-in-instrumentation-label-enabled` instead, which is user facing.

Reply:
I will remove the mention of `server.Config.ReportGRPCCodesInInstrumentationLabel`, but I wouldn't mention `-server.report-grpc-codes-in-instrumentation-label-enabled`. The reason for that is that even if a client set `-server.report-grpc-codes-in-instrumentation-label-enabled` to false, it will have no effect, and mentioning that CLI flag might look like that it is possible to change the defaults. WDYT @pracucci?